gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenSQLQueryTestSuite.scala

```diff
@@ -492,9 +307,9 @@ class GlutenSQLQueryTestSuite
     // If a test case is not in the test list, or it is in the ignore list, ignore this test case.
     if (
       !supportedList.exists(
-        t => testCase.name.toLowerCase(Locale.ROOT).contains(t.toLowerCase(Locale.ROOT))) ||
+        t => testCase.name.toLowerCase(Locale.ROOT).equals(t.toLowerCase(Locale.ROOT))) ||
       ignoreList.exists(
-        t => testCase.name.toLowerCase(Locale.ROOT).contains(t.toLowerCase(Locale.ROOT)))
+        t => testCase.name.toLowerCase(Locale.ROOT).equals(t.toLowerCase(Locale.ROOT)))
```

Comment:
With this change, I suppose we wil be missing some sql files under subdirectory. Can we specially handle group-by.sql only?

Reply:
Using `contains` can lead to some unexpected files being added. For example, `group-by.sql` was in the supported list but `udf/udf-group-by.sql` wasn't. But the latter one was also tested. If we are sure that one file is supported, it should be explicitly added to the supported list.

Reply:
Yes, that's just what I mean. With this change, we may be missing some tests for the files under subdirectory. There are some other sql files under subdirectory but share the same name with those in root directory.

Reply:
Can we keep using `contains` and exclude `udf/udf-group-by.sql`? If there are more test cases in `udf/udf-group-by.sql`, maybe we can merge them into the `group-by.sql` added by ourselves.
This can make sure other tests are not missing.

Reply:
Is it intended that we use `contains` to make `udf/udf-group-by.sql` a test? I think it would be better to explicitly add it to the supported list. Moreover, the current supported list inconsistently mixes some files specified with subdirectories and others with only the filename. It would be confusing for others who are trying to add new files to the list.

Reply:
The way to use `contains` derives from Spark SQLQueryTestSuite. It makes sense to me if we need to change it in Gluten test.